samples/net/wpanusb/overlay-mcr20a.conf

```diff
@@ -1,2 +1,4 @@
+CONFIG_NET_L2_ETHERNET=n
+CONFIG_SPI=y
```

Comment:
I don't think you need this one, it's already selected by mcr20a

Reply:
That is a problem, it is not selected. If you apply first sanitycheck patch you get error with "sanitycheck -t usb"